src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -8,23 +8,37 @@
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")

```

Comment:
`@Where` is marked as deprecated, so instead you can use `@SQLRestriction("is_deleted=false")`

Reply:
my bad